src/main/java/baseball/entity/BaseballGameStatus.java

```diff
@@ -0,0 +1,10 @@
+package baseball.entity;
+
+enum BaseballGameStatus {
+
+    PROGRESS, END;
+
+    public boolean isEnd() {
+        return this.equals(END);
+    }
+}
```

Comment:
inEnd에 대해서 아래처름 == 비교를 할 수 있었는데 euqals를 사용하신 이유가 있을까요?? 👍
```suggestion
enum BaseballGameStatus {

    PROGRESS, END;

    public boolean isEnd() {
        return this == END;
    }
}
```

Reply:
아무 생각없이 equals를 사용해왔는데요. 수찬님께서 말씀해주신 덕분에 ==과 무슨 차이가 있는지 공부해봤어요! 감사합니다. 🙇🏻‍♂️ 그 과정에서 좋은 글을 발견한 것 같아서 아래에 공유드립니다!

https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals

Reply:
좋은 아티클 공유 감사해요!!